khl/guild.py

```diff
@@ -161,6 +161,15 @@ async def list_user(self, channel: Channel) -> List[User]:
         users = await self.gate.exec_pagination_req(api.Guild.userList(guild_id=self.id, channel_id=channel.id))
         return [User(_gate_=self.gate, _lazy_loaded_=True, **i) for i in users]
 
+    async def fetch_joined_channel(self, user: User, page: int = 1, page_size: int = 50) -> List[PublicChannel]:
+        response = await self.gate.exec_req(api.ChannelUser.getJoinedChannel(page=page, page_size=page_size, guild_id=self.id, user_id=user.id))
```

Comment:
there is a native paged req helper: `gate.exec_pagination_req()` which may make this work easier

Reply:
FYI: `Guild.list_user()/fetch_roles()` is an usage example for `exec_pagination_req()`